src/main/java/org/dcsa/ebl/model/base/AbstractShippingInstruction.java

```diff
@@ -79,4 +79,8 @@
   @Size(max = 250)
   @Column("displayed_name_for_place_of_delivery")
   private String displayedNameForPlaceOfDelivery;
+
+  @Column("displayedNameForPlaceOfDelivery")
+  @Size(max = 250)
+  private String displayed_name_for_place_of_delivery;

```

Comment:
This looks like a duplicate (with reversed naming standards)

Reply:
Removed